R/rows.R

```diff
@@ -8,7 +8,17 @@
 #' When `in_place = TRUE` these verbs do not generate `SELECT` queries, but
 #' instead directly modify the underlying data using `INSERT`, `UPDATE`, or
 #' `DELETE` operators. This will require that you have write access to
-#' the database.
+#' the database: the connection needs permission to insert, modify or delete
+#' rows, but not to alter the structure of the table.
+#'
+#' The default, `in_place = FALSE`, generates equivalent lazy tables (using
+#' `SELECT` queries) that allow previewing the result without actually
+#' modifying the underlying table on the database.
+#'
+#' @details
+#' For `in_place = TRUE`, the `x` argument must refer to a table instantiated
+#' with [tbl()] or [compute()], not to a lazy query. The [remote_name()]
+#' function is used to determine the name of the table to be updated.
```

Comment:
Maybe move this into `@param x,y`? I think you could just say something like "In most cases, `x`, will be a table, not a lazy query". Since the point of `in_place = FALSE` is just to preview the change.